invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -127,6 +127,7 @@ pre.export.result {
 
 .invenio-page-body {
   padding-bottom: @defaultPadding;
+  padding: 1rem 0 1rem;
```

Comment:
I would suggest to use the variables, since we rely on the centralised values. Padding bottom can be also removed from above since your change is overriding it anyway.
```suggestion
  padding: @defaultPadding 0 @defaultPadding;
```

Reply:
LGTM, thanks.